code/modules/antagonists/heretic/heretic_curses.dm

```diff
@@ -0,0 +1,283 @@
+/*!
+ * Contains all the curses a heretic can cast using their upgraded codex
+ */
+
+
+/**
+ * No longer a subtype of knowledge, curses are now handed out via the codex morbus using blood at a rune
+ */
```

Comment:
I don't understand this comment, `No longer a subtype of knowledge` but it still is? And in any case you generally shouldn't leave comments about the past